packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, h, Prop } from '@stencil/core';
+import { EInputFieldType, IErrorMsgs } from './text-field.types';
+
+@Component({
+    tag: 'kv-text-field',
+    styleUrl: 'text-field.scss',
+    shadow: true,
+})
+export class TextField {
```

Comment:
We need to know when the value of the input is updated in parent component. So an event is necessary achieve this.